### DISCUSSION TITLE
Why do some entities perform better than others?

### DISCUSSION THREAD

**Discussion by venti07** [explosion/spaCy#11531] [Help: Model Advice]
I have trained different entities within a NER task. Among others, I used spaCY Tok2Vec, Stanford NER and BERT for this purpose.

The results show that BERT models perform best on average. However, certain entities (3/9) perform better on spaCy and Stanford NER. I am now looking for general reasons why spaCy and Stanford give better results than BERT. It would be nice if a few can give their thoughts on this.

**Comment by polm** [comment]
Debugging models is pretty hard (see #3052), and there aren't really general reasons that non-BERT models would be better; it depends a lot on the details. Can you give examples of the entities you're training on, both the ones that do better with BERT and without?
